src/pages/guides/troubleshooting/faq.md

```diff
@@ -35,3 +51,25 @@ Content publishing: For the same reason above, you have the ability to pre-defin
 ### My user base consists of students/minors. How can I be assured they get appropriate content?
 
 The embedded Adobe Express full editor can be setup to launch student safe content even before a student is signed into the embedded experience. More information about [Adobe & Student Privacy](https://www.adobe.com/privacy/student-policy.html).
+
+### How many domains can be added with a single API key for a service?
+
+You can add upto five domains with a single API key. Additionally, there is no limit on the number of subdomains that you can employ for these services.
+
+### How do I enable SSO for my service?
+
+To enable Single Sign-On (SSO) for your service, when using Adobe express Embed SDK, you need to set it up using the Adobe admin console. 
+
+Learn to use the Adobe Admin Console to set up [Single Sign-On (SSO)](https://helpx.adobe.com/in/enterprise/using/sso-overview.html) with your Identity provider and troubleshoot setup-related problems.
+
+### Would Adobe need to approve the integrations before my users can start using it?
+
+Your users can use the integrations, but, we will display a red banner indicating, `The experience powred by Adobe Express has not been approved yet`. Once we review and approve your app, we will remove this banner. 
+
+### What is the versioning strategy for the SDK?
+
+The SDK follows a continous release model, with newer versions including bug fixes and feature enhancements. We are no longer approving integrations using using SDK V1, V2, or V3.
```

Comment:
Suggestion, We can add a line - Latest changes, bug fixes  can be tracked via changelog( https://developer.adobe.com/express/embed-sdk/docs/guides/changelog/) doc.